bulk/file.go

```diff
@@ -63,12 +74,10 @@ func (f *File) IsChangedLocal(ignoreDeleted bool) bool {
 		return !ignoreDeleted
 	}
 
-	// Round-trip to get consistent formatting. This is inefficient but a much
-	// nicer experience for people with auto-formatters set up in their editor
-	// or who may try to undo changes and get the formatting slightly off.
-	var tmp any
-	json.Unmarshal(b, &tmp)
-	b, _ = cli.MarshalShort("json", true, tmp)
+	b, err = reformat(b)
+	if err != nil {
+		return false
```

Comment:
Should the error be printed before returning false?  Won't this swallow errors from MarshalShort?